src/ReactiveUI/Bindings/Command/CommandBinderImplementation.cs

```diff
@@ -64,7 +64,7 @@ public class CommandBinderImplementation : ICommandBinderImplementation
 
             var bindingDisposable = BindCommandInternal(source, view, controlExpression, Observable.Defer(() => Observable.Return(withParameter())), toEvent ?? string.Empty, cmd =>
             {
-                if (!(cmd is IReactiveCommand rc))
+                if (cmd is not IReactiveCommand rc)
```

Comment:
Similar here for a test if ts possible I guess.

Reply:
Added tests for IReactiveCommand need to add further tests for ICommand, Found bug in the Func<T> paramater overload where the type is returned as a Unit instead of T